src/iexamine.cpp

```diff
@@ -2020,7 +2020,7 @@ void iexamine::kiln_empty(player &p, const tripoint &examp)
     } else {
         add_msg( _( "This kiln contains %s %s of material, and is ready to be fired." ),
                  format_volume( total_volume ), volume_units_abbr() );
-        g->draw_sidebar_messages(); // flush messages before popup
+        // g->draw_sidebar_messages(); // flush messages before popup
```

Comment:
commented out calls to `g->draw_sidebar_messages()` should be removed here, and 6 more times below in this file